### ISSUE TITLE
Error for CONV1D:  Layer not supported for visualizing

### ISSUE THREAD

**Issue by User2587** [Prodicode/ann-visualizer#28]
I'm getting the error for CNN 1 Dimension, below is the model I have created :
        model = Sequential()
        model.add(Conv1D(31, 31, padding='same', input_shape=(992, 2)))  # ignore -1
        model.add(BatchNormalization())
        model.add(LeakyReLU())
        model.add(MaxPooling1D(1,1, padding='same'))

        model.add(Conv1D(992, 5, padding='same'))
        model.add(BatchNormalization())
        model.add(LeakyReLU())
        model.add(MaxPooling1D(1,1, padding='same'))

        model.add(Flatten())
        model.add(Dense(1984))
        model.add(BatchNormalization())
        model.add(Activation("tanh"))
        model.add(Dropout(0.5))

        model.add(Dense(len(CATEGORIES)))
        model.add(BatchNormalization())
        model.add(Activation("softmax"))
